src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -3050,11 +3050,10 @@ public enum ChillingReign implements LogicCardInfo {
         };
       case AVERY_130:
         return supporter (this) {
-          text "Draw 3 cards. Then, your opponent discards their Benched Pokémon until they have 3 Benched Pokémon. You may play only 1 Supporter card during your turn."
+          text "Draw 3 cards. If you drew any cards in this way, your opponent discards Pokémon from their Bench until they have 3. You may play only 1 Supporter card during your turn."
           def eff
           onPlay {
             draw 3
-
             if ( opp.bench.size() > 3 && confirm("Discard 3 cards to force both players to discard their Benched Pokémon until they have 3 Benched Pokémon?") ) {
```

Comment:
Why discard 3 cards? And a confirm in general, should only check for enough bench for discarding.

The entire onPlay closure should be inside an initial `if (my.deck)`, so eventual "use this supporter's effect as this attack's effect" cards don't break ~~even if right now they'll still break~~ (say, the opponent has no draw but gets to copy the effect)

Reply:
Can't add a comment for that line, but the playRequirement should be `my.deck`, no or. From japanese FAQs on the card:

* If you have no deck you can't play the card, even if there's more than 3 Benched Pokémon on your opponent's side.
* If you have cards in deck and the opponent has 2 benched Pokémon, you'd draw and finish the effect.

Reply:
I left the text alone as I don't know how to fix this card. Not that proficient enough with this stuff so I left it for someone else who knows better to fix that.

Reply:
Ahh alright my bad! 😅